src/Datadog.Trace/Tracer.cs

```diff
@@ -372,6 +372,13 @@ public Span StartSpan(string operationName, ISpanContext parent = null, string s
             return StartSpan(operationName, tags: null, parent, serviceName, startTime, ignoreActiveScope, spanId: null);
         }
 
+        /// <summary>
+        /// Forces the tracer to immediately flush pending traces and send them to the agent.
+        /// To be called when the appdomain or the process is about to be killed in a non-graceful way.
+        /// </summary>
+        /// <returns>Task used to track the async flush operation</returns>
+        public Task ForceFlush() => FlushAsync();
```

Comment:
Should this be `ForceFlushAsync`? Given we have the `async` suffix elsewhere.

Reply:
Right